pkg/pod/pod_test.go

```diff
@@ -394,6 +398,76 @@ func TestMakePod(t *testing.T) {
 			}},
 			Volumes: append(implicitVolumes, toolsVolume, downwardVolume),
 		},
+	}, {
+		desc: "sidecar container with script",
+		ts: v1alpha1.TaskSpec{
+			Steps: []v1alpha1.Step{{Container: corev1.Container{
+				Name:    "primary-name",
+				Image:   "primary-image",
+				Command: []string{"cmd"}, // avoid entrypoint lookup.
+			}}},
+			Sidecars: []v1alpha1.Sidecar{
+				{
+					Container: corev1.Container{
```

Comment:
Suggest collapsing these brackets to match format with the Step on line 404.  Similarly for the init container on line 422.